.github/workflows/ci.yml

```diff
@@ -266,3 +266,26 @@ jobs:
         with:
           name: notebook-outputs
           path: out
+  ts-lint:
+    name: Typescript lint check
+    runs-on: ubuntu-20.04
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          # Do we want check multiple versions of Node.js?
```

Comment:
no, we don't, this is development only, we control the dev env. If the nodejs side would be something that we think is going to be used as a library, then we would - but that is not the case!